server/api/e2e/gql_document_test.go

```
@@ -0,0 +1,466 @@
+		if (r.URL.Path == "/api/graphql" || r.URL.Path == "api/graphql") && r.Method == "POST" {
+			var gqlRequest struct {
+				Query     string         `json:"query"`
+				Variables map[string]any `json:"variables"`
+			}
+
+			bodyBytes, err := io.ReadAll(r.Body)
+			if err != nil {
+				http.Error(w, "Failed to read request body", http.StatusBadRequest)
+				return
+			}
+			r.Body.Close()
```

golangci-lint (1.62.2)

Error return value of `r.Body.Close` is not checked

(errcheck)